content/docs/quickstart/sqlalchemy.md

```diff
@@ -21,21 +31,24 @@ CONNSTR = f'postgresql://{USERNAME}:{TOKEN}@{PROJECT_ID}.cloud.neon.tech/main'
 engine = create_engine(CONNSTR)
 ```
 
-References:
+## References
 
 - [Establishing Connectivity - the Engine](https://docs.sqlalchemy.org/en/14/tutorial/engine.html)
 - [Connecting to PostgreSQL with SQLAlchemy](https://docs.sqlalchemy.org/en/14/core/engines.html#postgresql)
 
-### Using from python + psycopg2
+## Step 2 — Using from Python + psycopg2
```

Comment:
I think it's not actually the second step, but a separate way of using Neon altogether, which should live in a separate file. Sqlalchemy should work as above.

Reply:
i have removed the numbered steps and left headings level 2 instead